vars/dockerExecute.groovy

```diff
@@ -58,7 +58,9 @@ import groovy.transform.Field
       */
     'dockerRegistryUrl',
     /**
+      * Non Kubernetes only:
       * The credentials for the docker registry of type username/password as we rely on docker jenkins plugin. If left empty, images are pulled anonymously.
+      * For Kubernetes cases, pass credentials via `additionalPodProperties` parameter.
```

Comment:
Proposal: 
can we have something like this?
For Kubernetes cases, pass **secrets name** via `additionalPodProperties` parameter (**secrets should be presented on K8s**)

Reply:
Sure clarifies it better , Ill change